deploy/helm/kuberhealthy/values.yaml

```diff
@@ -3,7 +3,7 @@
 # Declare variables to be passed into your templates.
 
 checkReaper:
-  maxCheckPods: 4 # the maximum number of check pods in Completed state before being reaped
+  maxCompletedCheckPods: 4 # Maximum number of check pods in Completed state before being reaped, excludes pod from the most recent check run. If not set or set to 0, only the most recent check pod will remain.
```

Comment:
We might want to allow people to have no history at all.  In the case that the user configures a `0` here, we probably want to delete anything that is in the state `Completed` or `Error` immediately.

'most recent' in the comment leads me to believe that we always keep one `Completed` pod around, even if one is running.  I don't think that is required.  We could let users have no history at all if they wish.

Reply:
Sorry for getting involved here like this. But I do agree with @integrii , it will be best if we leave no pods around if we set maxCompletedCheckPods. But may be we still leave 1 error pod around for 'failedPodCleanupDuration' time for debugging the issue in case of Errors.

Reply:
@AshutoshNirkhe, not at all and thanks for your input! makes sense 👍  will update to adjust for this